addons/binding/org.openhab.binding.evohome/src/main/java/org/openhab/binding/evohome/internal/api/EvohomeApiClient.java

```diff
@@ -17,4 +18,5 @@
 
     ControlSystem getControlSystem(int id);
 
-}
\ No newline at end of file
+    ZoneStatus getHeatingZone(int locationId, int zoneId);
```

Comment:
I see you're using the V2 data models in the interface. I'm fine with that, but it does kinda mitigate the use of an interface. Since I don't expect it to change soon, I'd say, drop it and just use the V2 API directly. If it does change later on an interface is easily refactored. Changing data models is where the bulk of the work is, but, again, I don't expect it to change soon or at all.